.github/workflows/compare.yml

```diff
@@ -0,0 +1,28 @@
+name: Compare index and readme
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
Not crazy well versed in GitHub actions, but will this not update the docs when there is a PR that hasn't been merged? If so, is this what we want because the docs may not have been approved?

Reply:
This action is just so that we make sure both README and docs/index.md are identical on every PR to main. We don't really automate the update of either of the two files as there can be problems with it.

Reply:
Oh okay sweet @jsemearns looks great!